src/appModel.ts

```diff
@@ -293,6 +302,25 @@ export class AppModel {
         return SassHelper.targetCssFormat(outputStyleFormat);
     }
 
+    /**
+     * Autoprefixes CSS properties
+     * 
+     * @param css String representation of CSS to transform 
+     * @param target What browsers to be targeted, as supported by [Browserslist](https://github.com/ai/browserslist)
+     */
+    private async autoprefix(css: string, target: string): Promise<string> {
+        let result = '';
+
```

Comment:
But `target` & `result` is not used anywhere.

Reply:
what have I been doing

Thanks for reviewing!